### DISCUSSION TITLE
When requesting personalized content, should you specify the fetch cache as 'no-cache'?

### DISCUSSION THREAD

**Discussion by Eunkyung-Son** [vercel/next.js#62374] [Help] [answered]
### Summary

When making an API request with an Authorization header, I include a cookie-based authentication token. In this case, if I cache personalized content (cache: 'force-cache'), I believe that all users would receive the same content based on the cached content at build time. Is it correct to request personalized content with 'no-cache'? If so, are there any best practices for caching personalized content?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The docs do talk about this:

<img width="727" alt="Screenshot 2024-02-22 at 09 24 28" src="https://github.com/vercel/next.js/assets/21013447/bee00028-3011-4d67-9d20-75e61400c444">

I guess you are interested in:

> - The fetch request uses Authorization or Cookie headers and there's an uncached request above it in the component tree.

Perhaps it is a good idea to be defensive anyway? and use `no-store`?

> The no-cache option behaves the same way as no-store in Next.js.
